tests/testthat/test-mainTests.R

```diff
@@ -129,28 +123,28 @@ test_that("not_table_tool", {
   expect_equal(testIndividualTest(pathStart, "not_table_tool"), "ANCILLARY")
 })
 
-# old_la_code -------------------------------------------------------------------------------------------------------
+# la_combinations -------------------------------------------------------------------------------------------------------
 
-test_that("old_la_code", {
-  expect_equal(testIndividualTest(pathStart, "old_la_code"), "FAIL")
+test_that("la_combinations", {
+  expect_equal(testIndividualTest(pathStart, "la_combinations"), "FAIL")
 })
 
-# region_code -------------------------------------------------------------------------------------------------------
+# region_combinations -------------------------------------------------------------------------------------------------------
 
-test_that("region_code", {
-  expect_equal(testIndividualTest(pathStart, "region_code"), "FAIL")
+test_that("region_combinations", {
+  expect_equal(testIndividualTest(pathStart, "region_combinations"), "FAIL")
 })
 
-# country_code -------------------------------------------------------------------------------------------------------
+# country_combinations -------------------------------------------------------------------------------------------------------
 
-test_that("country_code", {
-  expect_equal(testIndividualTest(pathStart, "country_code"), "FAIL")
+test_that("country_combinations", {
+  expect_equal(testIndividualTest(pathStart, "country_combinations"), "FAIL")
 })
 
-# country_code -------------------------------------------------------------------------------------------------------
+# country_combinations -------------------------------------------------------------------------------------------------------
 
-test_that("country_code", {
-  expect_equal(testIndividualTest(pathStart, "country_code"), "FAIL")
+test_that("country_combinations", {
```

Comment:
have you duplicated this test? not sure why it's there twice

Reply:
Hmm I didn't add the second one, I just updated the check name. It does look like a duplicate though so I'll remove.